experimenter/experimenter/targeting/constants.py

```diff
@@ -38,6 +38,19 @@ def __post_init__(self):
 CORE_ACTIVE_USERS_TARGETING = "'{event}'|eventCountNonZero('Days', 28, 0) >= 21"
 RECENTLY_LOGGED_IN_USERS_TARGETING = "'{event}'|eventCountNonZero('Weeks', 12, 0) >= 1"
 
+# The following indicate whether the user accepted the terms of use in one of our initial phases of experimentation/rollout
+ACCEPTED_TOU_IN_NIMBUS_EXPERIMENT = "'datareporting.policy.dataSubmissionPolicyAcceptedVersion'|preferenceValue == 3"
+# Accepted in 100% on-train rollout in 139 release (after May 27, 6am PT timestamp), or in earlier partial rollout indicated by presence of on-train rollout population value
+ACCEPTED_TOU_IN_ON_TRAIN_ROLLOUT = "(('browser.preonboarding.enrolledInOnTrainRollout'|preferenceValue && 'datareporting.policy.dataSubmissionPolicyAcceptedVersion'|preferenceValue == 2) && ('datareporting.policy.dataSubmissionPolicyNotifiedTime'|preferenceValue >= "1748350800000" || 'browser.preonboarding.onTrainRolloutPopulation'|preferenceValue))"
```

Comment:
The current thing fails to parse because of the "174..." inside the larger quoted string.  Switching the inner `"` symbols to `"\` should fix this.